src/solver/optimisation/opt_optimisation_lineaire.cpp

```diff
@@ -110,8 +110,7 @@ bool runWeeklyOptimization(const OptimizationOptions& options,
                                  writer))
             return false;
 
-        if (problemeHebdo->ExportMPS != Data::mpsExportStatus::NO_EXPORT
-            || problemeHebdo->Expansion)
+        if (problemeHebdo->ExportMPS != Data::mpsExportStatus::NO_EXPORT)
```

Comment:
SimulationMode shouldn't be checked also ?

Reply:
No, this is the goal of this PR. To de-correlate the MPS export & the simulation mode